internal/generate/clusterserviceversion/clusterserviceversion_updaters.go

```diff
@@ -292,18 +305,25 @@ func validatingToWebhookDescription(webhook admissionregv1.ValidatingWebhook) op
 		TimeoutSeconds:          webhook.TimeoutSeconds,
 		AdmissionReviewVersions: webhook.AdmissionReviewVersions,
 	}
+	if len(description.AdmissionReviewVersions) == 0 {
+		description.AdmissionReviewVersions = defaultAdmissionReviewVersions
+	}
+
 	if serviceRef := webhook.ClientConfig.Service; serviceRef != nil {
 		if serviceRef.Port != nil {
 			description.ContainerPort = *serviceRef.Port
 		}
```

Comment:
Also, if serviceRef.Port == nil and description.ContainerPort is not set, can we default it to 9443 which is used in case of cert-manager. It is a required field which is to be defined in WebhookDescription. Also, since 443 (which olm defaults to) is restricted to root user, I suppose setting it to 9443 is a better option?

Reply:
Will do in a follow-up.